### PR TITLE
Use relative URL for `https://clerk.com` links

### PR DESCRIPTION
**Pull request by bradlc**
This PR replaces absolute `https://clerk.com` links with relative URLs, e.g.

```diff
- See the [pricing](https://clerk.com/pricing) page for more information.
+ See the [pricing](/pricing) page for more information.
```

This prevents the links from being considered external URLs and opening in a new tab. It also ensures that links work in non-production environments.

> [!NOTE]
> If you ever want to link to an internal page but you _do_ want it to open in a new tab, you can [annotate](https://npm.im/mdx-annotations) the link and set the [`target` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target) to `_blank`:
>
> ```mdx
> See the [pricing](/pricing){{ target: '_blank' }} page for more information.
> ```

### Questions

- Links to the backend/frontend API references are kept as absolute URLs for now since these only work on clerk.com anyway. Should these continue to open in a new tab?
- Should non-docs pages such as Pricing and Contact open in a new tab?

### CODE REVIEW

**Comment by alexisintech** [comment]
most of the time, this is done intentionally. so if it takes the user away from docs, its an external link (/pricing, /contact, and for bapi and fapi docs). 
can we do a script that instead of updating all these links, adds the {{ target: '_blank' }} syntax to them
and then i will go in manually and ensure that we want all of these as external

**Comment by bradlc** [comment]
@alexisintech gotcha, makes sense! I've updated the PR to remove `https://clerk.com` and add `target: '_blank'`

`TutorialHero` doesn't currently support opening links in a new tab. Do you think that would be useful?

File: docs/deployments/exporting-users.mdx [hunk 1]
```diff
@@ -7,10 +7,10 @@ description: Learn how to export user's data from your Clerk application.
 
 ## Access user data in the backend
 
-By using the [GetUserList](https://clerk.com/docs/reference/backend-api/tag/Users#operation/GetUserList) endpoint of Clerk's Backend API, you can programmatically export your user's data safely.
+By using the [GetUserList](/docs/reference/backend-api/tag/Users#operation/GetUserList){{ target: '_blank' }} endpoint of Clerk's Backend API, you can programmatically export your user's data safely.
```
Comment (alexisintech) [inline docs/deployments/exporting-users.mdx:10]:
```suggestion
By using Clerk's [`GetUserList`](/docs/reference/backend-api/tag/Users#operation/GetUserList){{ target: '_blank' }} Backend API endpoint, you can programmatically export your user's data safely.
```

File: docs/deployments/migrate-from-cognito.mdx [hunk 2]
```diff
@@ -72,7 +72,7 @@ The provided script below lists your Cognito user pool users, calls
 > [!CAUTION]
 > As usual, rate limits apply to the `CreateUser` endpoint. [^1]
 
-[^1]: CreateUser has a rate limit rule of 20 requests per 10 seconds. [https://clerk.com/docs/reference/backend-api/tag/Users#operation/CreateUser](https://clerk.com/docs/reference/backend-api/tag/Users#operation/CreateUser)
+[^1]: CreateUser has a rate limit rule of 20 requests per 10 seconds. [https://clerk.com/docs/reference/backend-api/tag/Users#operation/CreateUser](/docs/reference/backend-api/tag/Users#operation/CreateUser){{ target: '_blank' }}
```
Comment (alexisintech) [inline docs/deployments/migrate-from-cognito.mdx:75]:
```suggestion
[^1]: [`CreateUser`](/docs/reference/backend-api/tag/Users#operation/CreateUser){{ target: '_blank' }} has a rate limit rule of 20 requests per 10 seconds.
```

File: docs/deployments/migrate-from-cognito.mdx [hunk 3]
```diff
@@ -202,8 +202,8 @@ you will be able to validate the migration behavior.
 
   Up until this point, you were possibly using the Cognito hosted UI for your application’s user sign-in.
   With the [one-time upload](#one-time-upload) out of the way and integration validated, you should be ready to
-  update your application to use Clerk’s managed [Account Portal](https://clerk.com/docs/account-portal/overview)
-  or the [`<SignIn />`](https://clerk.com/docs/components/authentication/sign-in) component.
+  update your application to use Clerk’s managed [Account Portal](/docs/account-portal/overview){{ target: '_blank' }}
+  or the [`<SignIn />`](/docs/components/authentication/sign-in){{ target: '_blank' }} component.
```
Comment (alexisintech) [inline docs/deployments/migrate-from-cognito.mdx:206]:
```suggestion
  update your application to use Clerk’s managed [Account Portal](/docs/account-portal/overview) or the [`<SignIn />`](/docs/components/authentication/sign-in) component.
```

File: docs/deployments/migrate-overview.mdx [hunk 4]
```diff
@@ -14,7 +14,7 @@ Each of these have trade-offs you'll need to consider on behalf of for your appl
 
 ## Basic export / import
 
-With basic export / import, you are taking an export from your previous tool and importing data into Clerk. The most common way to handle this is by making use of the [Backend API](/docs/reference/backend-api/tag/Users#operation/CreateUser) and its `CreateUser` API endpoint. It's important to note that the `CreateUser` endpoint is rate limited. Please review our [Rate Limits](https://clerk.com/docs/backend-requests/resources/rate-limits#backend-api-requests) page for details.
+With basic export / import, you are taking an export from your previous tool and importing data into Clerk. The most common way to handle this is by making use of the [Backend API](/docs/reference/backend-api/tag/Users#operation/CreateUser) and its `CreateUser` API endpoint. It's important to note that the `CreateUser` endpoint is rate limited. Please review our [Rate Limits](/docs/backend-requests/resources/rate-limits#backend-api-requests){{ target: '_blank' }} page for details.
```
Comment (alexisintech) [inline docs/deployments/migrate-overview.mdx:17]:
```suggestion
With basic export / import, you are taking an export from your previous tool and importing data into Clerk. The most common way to handle this is by making use of Clerk's [`CreateUser`](/docs/reference/backend-api/tag/Users#operation/CreateUser){{ target: '_blank' }} Backend API endpoint. It's important to note that the `CreateUser` endpoint is rate limited. Please review our [Rate Limits](/docs/backend-requests/resources/rate-limits#backend-api-requests) page for more details.
```

File: docs/deployments/migrate-overview.mdx [hunk 5]
```diff
@@ -68,7 +68,7 @@ A trickle migration is great for upgrading active users and sessions to take adv
 
 ## Migration tools
 
-To aid in basic migrations, Clerk provides an open-source script that takes a JSON file as input, containing a list of users, and creates a user in Clerk using Clerk's backend API. The script respects Clerk's [backend rate limits](https://clerk.com/docs/backend-requests/resources/rate-limits#backend-api-requests) and gracefully handles errors. We suggest you customize the [Zod schema](https://github.com/clerk/migration-script/blob/main/index.ts#L25-L43) to your application's needs.
+To aid in basic migrations, Clerk provides an open-source script that takes a JSON file as input, containing a list of users, and creates a user in Clerk using Clerk's backend API. The script respects Clerk's [backend rate limits](/docs/backend-requests/resources/rate-limits#backend-api-requests){{ target: '_blank' }} and gracefully handles errors. We suggest you customize the [Zod schema](https://github.com/clerk/migration-script/blob/main/index.ts#L25-L43) to your application's needs.
```
Comment (alexisintech) [inline docs/deployments/migrate-overview.mdx:71]:
```suggestion
To aid in basic migrations, Clerk provides an open-source script that takes a JSON file as input, containing a list of users, and creates a user in Clerk using Clerk's Backend API. The script respects Clerk's [backend rate limits](/docs/backend-requests/resources/rate-limits#backend-api-requests)and gracefully handles errors. We suggest you customize the [Zod schema](https://github.com/clerk/migration-script/blob/main/index.ts#L25-L43){{ target: '_blank' }} to your application's needs.
```

File: docs/guides/add-onboarding-flow.mdx [hunk 6]
```diff
@@ -64,7 +64,7 @@ declare global {
 
 ## Configure your Middleware to read session data
 
-Clerk's [`clerkMiddleware()`](https://clerk.com/docs/references/nextjs/clerk-middleware) allows you to configure access to your routes with fine grained control. It also allows you to retrieve claims directly from the session and redirect your user accordingly.
+Clerk's [`clerkMiddleware()`](/docs/references/nextjs/clerk-middleware){{ target: '_blank' }} allows you to configure access to your routes with fine grained control. It also allows you to retrieve claims directly from the session and redirect your user accordingly.
```
Comment (alexisintech) [inline docs/guides/add-onboarding-flow.mdx:67]:
```suggestion
Clerk's [`clerkMiddleware()`](/docs/references/nextjs/clerk-middleware) allows you to configure access to your routes with fine grained control. It also allows you to retrieve claims directly from the session and redirect your user accordingly.
```

File: docs/upgrade-guides/core-2/javascript.mdx [hunk 7]
```diff
@@ -508,23 +508,23 @@ As part of this major version, a number of previously deprecated props, arugment
   </AccordionPanel>
 
   <AccordionPanel>
-    The experimental property `Clerk.experimental_canUseCaptcha` has been removed. There is no replacement for this functionality currently. If this is problematic for your application, please [reach out to our support team](https://clerk.com/contact/support)!
+    The experimental property `Clerk.experimental_canUseCaptcha` has been removed. There is no replacement for this functionality currently. If this is problematic for your application, please [reach out to our support team](/contact/support){{ target: '_blank' }}!
```
Comment (alexisintech) [inline docs/upgrade-guides/core-2/javascript.mdx:511]:
```suggestion
    The experimental property `Clerk.experimental_canUseCaptcha` has been removed. There is no replacement for this functionality currently. If this is problematic for your application, please [reach out to Clerk support](/contact/support){{ target: '_blank' }}!
```

File: docs/upgrade-guides/core-2/javascript.mdx [hunk 8]
```diff
@@ -508,23 +508,23 @@ As part of this major version, a number of previously deprecated props, arugment
...
   </AccordionPanel>
 
   <AccordionPanel>
-    The experimental property `Clerk.experimental_captchaURL` has been removed. There is no replacement for this functionality currently. If this is problematic for your application, please [reach out to our support team](https://clerk.com/contact/support)!
+    The experimental property `Clerk.experimental_captchaURL` has been removed. There is no replacement for this functionality currently. If this is problematic for your application, please [reach out to our support team](/contact/support){{ target: '_blank' }}!
```
Comment (alexisintech) [inline docs/upgrade-guides/core-2/javascript.mdx:515]:
```suggestion
    The experimental property `Clerk.experimental_captchaURL` has been removed. There is no replacement for this functionality currently. If this is problematic for your application, please [reach out to Clerk support](/contact/support){{ target: '_blank' }}!
```

File: docs/upgrade-guides/core-2/javascript.mdx [hunk 9]
```diff
@@ -508,23 +508,23 @@ As part of this major version, a number of previously deprecated props, arugment
...
   </AccordionPanel>
 
   <AccordionPanel>
-    The experimental property `Clerk.experimental_captchaSiteKey` has been removed. There is no replacement for this functionality currently. If this is problematic for your application, please [reach out to our support team](https://clerk.com/contact/support)!
+    The experimental property `Clerk.experimental_captchaSiteKey` has been removed. There is no replacement for this functionality currently. If this is problematic for your application, please [reach out to our support team](/contact/support){{ target: '_blank' }}!
```
Comment (alexisintech) [inline docs/upgrade-guides/core-2/javascript.mdx:519]:
```suggestion
    The experimental property `Clerk.experimental_captchaSiteKey` has been removed. There is no replacement for this functionality currently. If this is problematic for your application, please [reach out to Clerk support](/contact/support){{ target: '_blank' }}!
```

File: docs/upgrade-guides/core-2/javascript.mdx [hunk 10]
```diff
@@ -647,7 +647,7 @@ As part of this major version, a number of previously deprecated props, arugment
     To set the `afterSignUpUrl` or `afterSignInUrl`, you can:
 
     - If not using a react-based SDK, pass the values into `Clerk.load` as such: `Clerk.load({ afterSignUpUrl: 'x', afterSignInUrl: 'y' })`
-    - If using a react-based SDK, [pass the desired values into ClerkProvider](https://clerk.com/docs/components/clerk-provider#properties)
+    - If using a react-based SDK, [pass the desired values into ClerkProvider](/docs/components/clerk-provider#properties){{ target: '_blank' }}
```
Comment (alexisintech) [inline docs/upgrade-guides/core-2/javascript.mdx:650]:
```suggestion
    - If using a React-based SDK, pass the desired values into [`<ClerkProvider>`](/docs/components/clerk-provider#properties).
```

File: docs/users/overview.mdx [hunk 11]
```diff
@@ -25,7 +25,7 @@ Clerk provides the prebuilt components [`<UserButton />`](/docs/components/user/
 
 If you are using React, the [React SDK](/docs/references/react/use-user) provides hooks to help manage user authentication and profile data.
 
-If Clerk's prebuilt components don't meet your specific needs or if you require more control over the logic, you can use React hooks and JavaScript methods to [rebuild existing Clerk flows.](https://clerk.com/docs/custom-flows/overview)
+If Clerk's prebuilt components don't meet your specific needs or if you require more control over the logic, you can use React hooks and JavaScript methods to [rebuild existing Clerk flows.](/docs/custom-flows/overview){{ target: '_blank' }}
```
Comment (alexisintech) [inline docs/users/overview.mdx:28]:
```suggestion
If Clerk's prebuilt components don't meet your specific needs or if you require more control over the logic, you can use React hooks and JavaScript methods to [rebuild existing Clerk flows.](/docs/custom-flows/overview)
```

File: docs/users/user-impersonation.mdx [hunk 12]
```diff
@@ -63,7 +63,7 @@ Once you've successfully created an actor token, you can use the `url` attribute
 
 The `url` attribute is a Clerk Frontend API URL that will use the token to sign out existing users and prepare the [`SignIn`](/docs/references/javascript/sign-in/sign-in) object for impersonation. You can directly visit the `url` provided in the response to consume the actor token.
 
-The [Clerk Frontend API](https://clerk.com/docs/reference/frontend-api/) will redirect you to the `/sign-in` page of your application, where the flow will continue by consuming the `__clerk_ticket` parameter.
+The [Clerk Frontend API](/docs/reference/frontend-api/){{ target: '_blank' }} will redirect you to the `/sign-in` page of your application, where the flow will continue by consuming the `__clerk_ticket` parameter.
```
Comment (alexisintech) [inline docs/users/user-impersonation.mdx:66]:
```suggestion
Clerk's [Frontend API](/docs/reference/frontend-api/){{ target: '_blank' }} will redirect you to the `/sign-in` page of your application, where the flow will continue by consuming the `__clerk_ticket` parameter.
```

**Review by alexisintech — APPROVED** [review]

**Review by alexisintech — APPROVED** [review]

**Pull request merged**